src/components/PeopleTable.tsx

```diff
@@ -1,4 +1,42 @@
-export const PeopleTable = () => {
+import React from 'react';
+import classNames from 'classnames';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+import { SortLink } from './SortLink';
+import { SortTypes } from '../types/SortTypes';
+
+interface Props {
+  people: Person[],
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { personSlug = '' } = useParams();
+  const [searchParams] = useSearchParams();
+
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+
+  const handleSortPeople = () => {
+    const sortPeople = [...people].sort((prevPerson, nextPerson) => {
+      switch (sort) {
+        case SortTypes.Name:
+        case SortTypes.Sex:
+          return prevPerson[sort].localeCompare(nextPerson[sort]);
+        case SortTypes.Born:
+        case SortTypes.Died:
+          return prevPerson[sort] - nextPerson[sort];
+        default: {
+          return 0;
+        }
```

Comment:
add empty line between case blocks (after return)

```suggestion
        default:
          return 0;
```